modules/calib3d/src/calibration_handeye.cpp

```diff
@@ -572,7 +598,7 @@ static void calibrateHandEyeAndreff(const std::vector<Mat>& Hg, const std::vecto
     R = R.reshape(1, 2, newSize);
     //Eq 15
     double det = determinant(R);
-    R = pow(sign_double(det) / abs(det), 1.0/3.0) * R;
+    R = cubic_root(sign_double(det) / abs(det)) * R;
```

Comment:
What is wrong with `cv:cubeRoot()`?

Reply:
> What is wrong with cv:cubeRoot()`?

Didn't know of this function.

For master, I still think it is better and cleaner to use C+11 `std::cbrt` since `cubeRoot` is only single-precision accuracy.
Feel free to close the master PR if it is easier to merge 3.4 in master.

---

The code does this:

![image](https://user-images.githubusercontent.com/8229425/90998062-7a368c00-e5c3-11ea-9508-cf18e37152e2.png)